featurebyte/service/working_schema.py

```diff
@@ -82,12 +83,18 @@ def __init__(self, user: Any, persistent: Persistent, celery: Celery, catalog_id
         self.online_store_table_version_service = OnlineStoreTableVersionService(
             user=user, persistent=persistent, catalog_id=catalog_id
         )
-        self.tile_manager_service = TileManagerService(
+        self.tile_scheduler_service = TileSchedulerService(
```

Comment:
side note: i've a separate PR to use the app_container for the data migration stuff as well, but will get back to it after i finish the target integration tests

Reply:
Thanks! Having that would be great 👍